versioned_docs/version-1.5/reference/manifest.mdx

```diff
@@ -155,37 +155,43 @@ A list of commands to deploy your development environment.
 
 It's usually a combination of `helm`, `kubectl`, and `okteto` commands.
 
-#### Deploy a Docker Compose file:
+#### Deploy a Helm chart
```

Comment:
Based on the text above, it reads better if we document helm before docker compose